docs/source/conf.py

```diff
@@ -20,43 +24,30 @@
     "sphinx.ext.autodoc",
     "sphinx.ext.napoleon",
     "sphinx.ext.viewcode",
+    "sphinx.ext.intersphinx",
+    "sphinx_substitution_extensions",
 ]
 
-templates_path = ["_templates"]
-exclude_patterns = ["**/installation_template.rst"]
+# Provide substitutions for common values
+rst_prolog = f"""
+.. |project| replace:: {project}
+.. |copyright| replace:: {copyright}
+.. |author| replace:: {author}
+.. |version| replace:: {version}
+.. |release| replace:: {release}
+"""
 
 rst_epilog = """
 .. |br| raw:: html
 
    <br />
 """
 
+intersphinx_mapping = {
+    "anndata": ("https://anndata.readthedocs.io/en/stable/", None),
+}
+
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 html_theme = "sphinx_rtd_theme"
-html_static_path = ["_static"]
-
-
-def substitute_installation_version():
```

Comment:
Is `sphinx_substitution_extensions` does this?

Reply:
Yep!